src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -124,6 +141,9 @@ export class CourseUpdateComponent implements OnInit {
                 onlineCourse: new FormControl(this.course.onlineCourse),
                 complaintsEnabled: new FormControl(this.complaintsEnabled),
                 requestMoreFeedbackEnabled: new FormControl(this.requestMoreFeedbackEnabled),
+                maxPoints: new FormControl(this.course.maxPoints, {
+                    validators: [Validators.min(1)],
```

Comment:
is this validator only active when the value is set?

Reply:
yes